src/elements/Button/ButtonGroup.js

```diff
@@ -0,0 +1,79 @@
+import cx from 'classnames'
+import React, { PropTypes } from 'react'
+
+import META from '../../utils/Meta'
+import * as sui from '../../utils/semanticUtils'
+import {
+  getUnhandledProps,
+  useKeyOnly,
+} from '../../utils/propUtils'
+
+/**
+ * Button.Group
+ */
+function ButtonGroup(props) {
+  const {
+    basic, className, children, color, icon, size, vertical, widths,
+  } = props
+
+  const classes = cx('sd-button-group', 'ui',
+    size,
+    color,
+    useKeyOnly(basic, 'basic'),
+    useKeyOnly(icon, 'icon'),
+    useKeyOnly(vertical, 'vertical'),
+    className,
+    widths,
+    'buttons'
+  )
+
+  const rest = getUnhandledProps(ButtonGroup, props)
+
+  return (
+    <div className={classes} {...rest}>
+      {children}
+    </div>
+  )
+}
+
+
+ButtonGroup._meta = {
+  library: META.library.semanticUI,
+  name: 'ButtonGroup',
+  parent: 'Button',
+  type: META.type.element,
+  props: {
+    color: sui.colors,
+    size: sui.sizes,
+    widths: sui.widths,
+  },
+}
+
+
+ButtonGroup.propTypes = {
+  /** A button group can be less pronounced */
+  basic: PropTypes.bool,
+
+  /** Class names for custom styling. */
+  className: PropTypes.string,
+
+  /** */
+  children: PropTypes.any,
+
+  /** Groups can have a shared color */
+  color: PropTypes.oneOf(ButtonGroup._meta.props.color),
+
+  /** Groups can be formatted as icons */
+  icon: PropTypes.bool,
+
+  /** A button can have different sizes */
+  size: PropTypes.oneOf(ButtonGroup._meta.props.size),
+
+  /** Groups can be formatted to appear vertically */
+  vertical: PropTypes.bool,
+
+  /** Groups can have their widths divided evenly */
+  widths: PropTypes.oneOf(ButtonGroup._meta.props.widths),
```

Comment:
I think the real intention here is to have buttons be equal width.  In the FormFields we have `evenlyDivided` which adds the number-word class to the group (ie `five fields`) by counting the field children.  [Evenly divided](http://semantic-ui.com/collections/form.html#evenly-divided) is the name of that section in the SUI docs.

I could see going with `equalWidth` here and counting the button children.  It is also consistent with the grid class `equal width` which scales column widths.

In order to not tie the user's hands should they want odd sized button groups, we could offer both `equalWidth` and `widths`.  Or, just use something like `widths='equal'`.